melt_czochralski_Dadzis/multilog_eln.archive.yaml

```diff
@@ -2,51 +2,51 @@ definitions:
   name: Electronic Lab Notebook
   sections:
     Data_processing:
-        base_sections: 
-        - ../upload/raw/base_classes.schema.archive.yaml#Task
-        quantities:
-          software:
-            type: str
-            m_annotations:
-              eln:
-                component: StringEditQuantity
-            description: Software used for logging
-          sampling_time:
-            type: np.float64
-            unit: ms
-            m_annotations:
-              eln:
-                component: NumberEditQuantity
-                defaultDisplayUnit: ms
-            description: Time between sampled points
-          image_time:
-            type: np.float64
-            unit: ms
-            m_annotations:
-              eln:
-                component: NumberEditQuantity
-                defaultDisplayUnit: ms
-            description: Time between recorded images
+      base_sections:
+      - ../upload/raw/base_classes.schema.archive.yaml#Task
+      quantities:
+        software:
+          type: str
+          m_annotations:
+            eln:
+              component: StringEditQuantity
+          description: Software used for logging
+        sampling_time:
+          type: np.float64
+          unit: ms
+          m_annotations:
+            eln:
+              component: NumberEditQuantity
+              defaultDisplayUnit: ms
+          description: Time between sampled points
+        image_time:
+          type: np.float64
+          unit: ms
+          m_annotations:
+            eln:
+              component: NumberEditQuantity
+              defaultDisplayUnit: ms
+          description: Time between recorded images
     My_nice_process:
```

Comment:
This was just a dummy example name,  are you changing it?

Reply:
I still need to think about how to use it, so I just kept it as-is. I'll update it later!

Reply:
Ok, nice